src/load_save.c

```diff
@@ -1,21 +1,36 @@
-#include "global.h"
+#include "global.h"    
 #include "gba/flash_internal.h"
 #include "load_save.h"
 #include "main.h"
 #include "pokemon.h"
 #include "rng.h"
-#include "malloc.h"
 
 extern void* gUnknown_0203CF5C;
 
 extern bool16 IdentifyFlash(void);
 extern void SetBagItemsPointers(void);
 extern void SetDecorationInventoriesPointers(void);
+extern void ApplyNewEncyprtionKeyToGameStats(u32 key);
+extern void ApplyNewEncyprtionKeyToBagItems(u32 newKey);
+extern void ApplyNewEncyprtionKeyToBagItems_(u32 key);
+extern void ApplyNewEncyprtionKeyToBerryPowder(u32 key);
```

Comment:
You should go ahead and fix all instances of "Encyprtion" to "Encryption" in this PR.